### PR TITLE
Add missing decorators

### PR DESCRIPTION
**Pull request by atd-schubert**

### CODE REVIEW

File: ts/tile-layer.directive.ts [hunk 1]
```diff
@@ -218,70 +218,72 @@ export class TileLayerDirective extends TileLayer implements OnDestroy  {
         return this.options.zIndex;
     }
 
-    set tileSize(val: Point) {
+    @Input() set tileSize(val: Point) {
+        (<TileLayerOptions>(<any>this).options).tileSize = val;
```
Comment (atd-schubert) [inline ts/tile-layer.directive.ts]: options as TileLayer options is now in the main declaration...

File: ts/tile-layer.directive.ts [hunk 2]
```diff
@@ -218,70 +218,72 @@ export class TileLayerDirective extends TileLayer implements OnDestroy  {
...
         this.options.tileSize = val;
     }
     get tileSize(): Point { // TODO: is this correct that it is always a Point?
         return (<Point>this.options.tileSize);
     }
 
-    set updateWhenIdle(val: boolean) {
+    @Input() set updateWhenIdle(val: boolean) {
+        (<TileLayerOptions>(<any>this).options).updateWhenIdle = val;
```
Comment (atd-schubert) [inline ts/tile-layer.directive.ts]: options as TileLayer options is now in the main declaration...

**Review by atd-schubert — COMMENTED** [review]
I should change this first...

**Review by OpenSteveMap — APPROVED** [review]
Approved...

**Pull request merged**